src/relative-time.js

```diff
@@ -187,8 +189,8 @@ export default class RelativeTime {
   }
 }
 
-function formatRelativeTime(value: number, unit: string): string {
-  const formatter = relativeFormatter()
+function formatRelativeTime(locale: string, value: number, unit: string): string {
+  const formatter = makeRelativeFormatter(locale, {numeric: 'auto'})()
```

Comment:
The formatter instance caching in `makeRelativeFormatter` isn't really effective since we invoke the factory function each time here. Maybe we should separate the factory from the caching and just call the factory here so it's clear it isn't attempting to cache it.

```js
// eslint-disable-next-line flowtype/no-weak-types
export function makeRelativeFormatter(options: any): () => ?any {
  let format
  return function() {
    if (format) return format
    format = makeRelativeFormat(options)
  }
}

export function makeRelativeFormat(options: any): ?Intl.RelativeTimeFormat {
  if ('Intl' in window && 'RelativeTimeFormat' in window.Intl) {
    try {
      // eslint-disable-next-line flowtype/no-flow-fix-me-comments
      // $FlowFixMe: missing RelativeTimeFormat type
      return new Intl.RelativeTimeFormat(undefined, options)
    } catch (e) {
      if (!(e instanceof RangeError)) {
        throw e
      }
    }
  }
}
```